app/config.py

```diff
@@ -276,6 +276,8 @@ def __init__(self, runtime_environment):
         self.script_chunk_size = int(os.getenv("SCRIPT_CHUNK_SIZE", "500"))
         self.rebuild_events_time_limit = int(os.getenv("REBUILD_EVENTS_TIME_LIMIT", "3600"))  # 1 hour
         self.sp_authorized_users = os.getenv("SP_AUTHORIZED_USERS", "tuser@redhat.com").split()
+        self.mq_db_batch_max_messages = int(os.getenv("MQ_DB_BATCH_MAX_MESSAGES", "50"))
```

Comment:
```suggestion
        self.mq_db_batch_max_messages = int(os.getenv("MQ_DB_BATCH_MAX_MESSAGES", "1"))
```
Nit, but to be consistent with the clowdapp file should this be `1`. Don't have to change it now, but noticed the inconsistency where a dev might have assumed the in code default might be used.